src/main/java/core/basesyntax/serviceimpl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
There should not be an empty line at the beginning of the class definition. Please remove the empty line to adhere to the coding standards.